docs/privacy-policy.md

```diff
@@ -0,0 +1,74 @@
+# Privacy Policy
+
+#### _Effective:_ February 8, 2022
+
+
+This policy document explains how the Operate First Data Science Community Meetup (“we” / “us” / “our”) collects, stores, and uses attendees’ personal information. If you wish to access any particular section of this Privacy Policy, use the list below to jump to that section.
+
+1. [Collection Of Information](#Collection-Of-Information)
+
+2. [Use Of Information](#Use-Of-Information)
+
+3. [Sharing Of Information](#Sharing-Of-Information)
+
+4. [Additional Details](#Additional-Details)
+
+5. [References](#References)
+
+6. [Contact Us](#Contact-Us)
+
+
+
+## Collection Of Information
+
+We collect information about you when you:
+
+1. Login using a google account;
+2. Use the google meet platform;
+3. Attend an event;
+4. Visit our website; and
+5. Communicate with us.
+
+We may also collect information, such as usage statistics, by using cookies, server logs, and similar technologies.
+
+
+## Use Of Information
+
+We use information about you to:
+
+1. Operate and promote our Community;
```

Comment:
What Does "operate" mean here? We use it to promote upcoming events, but I'm not sure about "operate"